it/router/RouterAuditFeature.scala

```diff
@@ -501,18 +591,85 @@ class RouterAuditFeature extends StubbedFeatureSpec with CommonStubs {
         HAS_SA_ENROLMENTS.key -> "false"
         ))
       val expectedTransactionName = "sent to business tax account"
-      verifyAuditEvent(auditEventStub, expectedReasons, expectedTransactionName, "bta-home-page-affinity-group-unavailable")
+      val expectedThrottlingDetails = ThrottlingDetails(
+        enabled = false,
+        percentage = "-",
+        throttled = false,
+        destinationUrlBeforeThrottling = "/business-account",
+        destinationNameBeforeThrottling = "business-tax-account"
+      )
+      verifyAuditEvent(auditEventStub, expectedReasons, expectedTransactionName, "bta-home-page-affinity-group-unavailable", expectedThrottlingDetails)
     }
   }
+}
+
+class RouterAuditFeatureWithThrottling extends StubbedFeatureSpec with CommonStubs with AuditTools {
+  override lazy val app = FakeApplication(additionalConfiguration = config + (
+    "throttling.enabled" -> true,
+    "throttling.locations.personal-tax-account-verify.percentageBeToThrottled" -> "100",
+    "throttling.locations.personal-tax-account-verify.fallback" -> "business-tax-account"
+    ))
+
+  scenario("a user supposed to be redirected to PTA should be throttled to BTA") {
+
+    Given("a user logged in through Verify supposed to go to PTA")
+    SessionUser(loggedInViaGateway = false).stubLoggedIn()
+
+    val auditEventStub = stubAuditEvent()
+
+    createStubs(BtaHomeStubPage)
+
+    When("the user hits the router")
+    go(RouterRootPath)
+
+    Then("the user should be routed to BTA Home Page")
+    on(BtaHomePage)
+
+    And("user is sent to BTA an audit event should be sent")
+    verify(postRequestedFor(urlMatching("^/write/audit.*$")))
+
+    And("the audit event raised should be the expected one")
+    val expectedReasons = toJson(emptyRoutingReasons + (IS_A_VERIFY_USER.key -> "true"))
+    val expectedTransactionName = "sent to business tax account"
+    val expectedThrottlingDetails = ThrottlingDetails(
+      enabled = true,
+      percentage = "100",
```

Comment:
this is the only test i can see where `percentage` is anything other than `"-"`. Should there be a test where `percentage` is eg. 50, and then tests where `ThrottlingDetails` contains both `true` and `false` for `throttled` ?